CoreScriptsRoot/CoreScripts/BubbleChat.lua

```diff
@@ -29,15 +29,17 @@ local BILLBOARD_MAX_HEIGHT = 500
 local ELIPSES = "..."
 local CchMaxChatMessageLength = 128 -- max chat message length, including null terminator and elipses.
 local CchMaxChatMessageLengthExclusive = CchMaxChatMessageLength - string.len(ELIPSES) - 1
+
+local NON_CORESCRIPT_MODE = false
```

Comment:
have you tested this script to see if it currently works in a non core script mode?